lib/src/base-styles/_nx-text-input.scss

```diff
@@ -5,33 +5,38 @@
  * distribution and is available at https://www.eclipse.org/legal/epl-2.0/.
  */
 
-.nx-text-input {
-  border: $nx-border-dark;
-  border-left-width: 4px;
-  border-radius: 0;
-  box-shadow: none;
+.nx-text-input__input {
+  border: 1px solid #d1d6f0;
+  border-radius: $nx-border-radius;
+  color: $nx-text-color-light;
+  font-size: $nx-font-size;
   height: $nx-form-element-height;
```

Comment:
Is there a reason this is explicit rather than relying on content line-height and padding?

Reply:
> Is there a reason this is explicit rather than relying on content line-height and padding?

Just because that's how it was done, but I removed the height (and associated variables and it's consistent with other elements now: https://github.com/sonatype/sonatype-react-shared-components/pull/57/commits/9e5655f8bd6cf083b80038a89ff85fa94fa8941d